deploy/namespace-operator.yaml

```diff
@@ -36,6 +36,16 @@ rules:
   verbs:
   - create
   - patch
+- apiGroups:
+  - seccomp-operator.k8s-sigs.io
+  resources:
+  - seccompprofiles
+  verbs:
+  - create
```

Comment:
Do we need create permissions on the `SeccompProfile` resource?

Reply:
Nope, will remove